utils/biospecimen.js

```diff
@@ -623,16 +623,24 @@ const biospecimenAPIs = async (req, res) => {
         }
         const requestData = req.body;
         if(Object.keys(requestData).length === 0 ) return res.status(400).json(getResponseJSON('Request body is empty!', 400));
-        const uuid = require('uuid');
-        const currentDate = new Date().toISOString();
-        requestData.id = uuid();
-        requestData.timeStamp = currentDate;
         const { addKitAssemblyData } = require('./firestore');
         const response = await addKitAssemblyData(requestData);
         if(!response) return res.status(404).json(getResponseJSON('ERROR!', 404));
         return res.status(200).json({message: `Success!`, code:200})
     }
 
+    else if(api == 'updateKitData'){
+        if(req.method !== 'POST') {
```

Comment:
```suggestion
        if (req.method !== 'POST') {
```

nit: spacing after `if` keyword